### PR TITLE
Husky

### PR DESCRIPTION
**Pull request by mkholjuraev**
# Description

Associated Jira ticket: # (issue)

This is nice to have. It introduces husky to run tests before each commit is created. 

# How to test the PR

Create any commit and observe that tests are run directly after it got created.

# Before the change

# After the change

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [ ] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run test && npm run test:ct && npm run lint
```
Comment (gkarat) [inline .husky/pre-commit:4]: Woah, do you really want to run component tests too? :-D folks will be crazy waiting for tests to complete. better just leave it on per-pull-request CI?

Comment (mkholjuraev) [reply .husky/pre-commit:4]: I think everyone runs component tests before pushing to CI. Usually what I do is run the tests to check if tests are passing, if not update tests&snapshots and then push to master.

Comment (mkholjuraev) [reply .husky/pre-commit:4]: With this, you will be sure that your PR does not fail tests&lint once and always after  you create a PR.

Comment (mkholjuraev) [reply .husky/pre-commit:4]: Also you can easly disable husky while creating a WIP commit with **-n** or **--no-verify**

Comment (gkarat) [reply .husky/pre-commit:4]: ok, I see, still sometimes you'd need to commit quickly, but the flag should help. lgtm 👍🏼

**Review by gkarat — APPROVED** [review]

**Pull request merged**